seqr_sample_qc.py

```diff
@@ -131,16 +160,15 @@ def get_all_sample_metadata(mt: hl.MatrixTable, build: int, data_type: str, data
     meta_ht = meta_ht.annotate(seqr_id=hl.if_else(hl.is_missing(meta_ht.seqr_id), meta_ht.SAMPLE, meta_ht.seqr_id))
 
     logger.info("Filtering to bi-allelic, high-callrate, common SNPs to calculate callrate...")
-    mt = filter_rows_for_qc(mt, min_af=0.001,
-                            min_callrate=0.99,
-                            bi_allelic_only=True,
-                            snv_only=True,
-                            apply_hard_filters=False,
-                            min_inbreeding_coeff_threshold=None,
-                            min_hardy_weinberg_threshold=None,
-                            )
-    callrate_ht = mt.select_cols(filtered_callrate=hl.agg.fraction(hl.is_defined(mt.GT))).cols()
-    meta_ht = meta_ht.annotate(**callrate_ht[meta_ht.key])
+    mt = get_qc_mt(mt,
+                   apply_hard_filters=False,
+                   filter_decoy=False,
+                   filter_segdup=False,
+                   min_inbreeding_coeff_threshold=None,
```

Comment:
So I was trying to keep this as close to the original pipeline as possible. I think changing the callrate filtering may be a separate PR since that will change pipeline results.